sys/mount_linux.go

```diff
@@ -60,37 +60,62 @@ func FMountat(dirfd uintptr, source, target, fstype string, flags uintptr, data
 	runtime.LockOSThread()
 	defer runtime.UnlockOSThread()
 
+	var pipefds [2]int
+	if err := syscall.Pipe(pipefds[:]); err != nil {
```

Comment:
can we use `syscall.Pipe2` with `O_CLOEXEC`?  If use old api `Pipe`, we have to use fcntl with `O_CLOEXEC` to make sure there is no fd leak.